data/io.elementary.wingpanel.bluetooth.appdata.xml.in

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 elementary, Inc. <contact@elementary.io> -->
+<component type="addon">
+  <id>io.elementary.wingpanel.bluetooth</id>
+  <extends>io.elementary.wingpanel</extends>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-2.1</project_license>
+  <name>Wingpanel Bluetooth Indicator</name>
+  <summary>Provides an indicator in Wingpanel for managing Bluetooth connections</summary>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/elementary/wingpanel-indicator-bluetooth/master/data/screenshot.png</image>
+    </screenshot>
+  </screenshots>
+  <provides>
+    <library>libbluetooth.so</library>
```

Comment:
I don't think that we want this as it is not really a library

Reply:
I agree, it's more of an extension specific for Wingpanel, but technically it's providing this file and it will be installed under a public accessible library path, conform the AppStream specification. Any of the other `<provides>` child tags would not match the spec, I think

Reply:
> The provides tag and its children describe the public interfaces this application provides. A public interface can be anything which other applications, which are not part of the upstream project, can access or reference. This includes binaries and libraries. Private interfaces should never be added to a provides tag. 

Ok, I think we can consider this a private library and therefore leave it out, while adhering to the spec (I should've read it more closely :sweat_smile:)